cds_ils/importer/providers/safari/rules/document.py

```diff
@@ -96,12 +96,29 @@ def eitem(self, key, value):
 
 @model.over("identifiers", "^020__")
 @filter_list_values
-def identifiers(self, key, value):
+def print_identifiers(self, key, value):
     """Translate identifiers."""
     _identifiers = self.get("identifiers", [])
     isbn = {
         "scheme": "ISBN",
         "value": clean_val("z", value, str, req=True),
+        "material": "PRINT_VERSION"
+    }
+    if isbn not in _identifiers:
+        _identifiers.append(isbn)
+    return _identifiers
+
+
+@model.over("identifiers", "^0248_")
```

Comment:
in the issue was ind1=7 is 0248 intentional?
```suggestion
@model.over("identifiers", "^0247_")
```

Reply:
that was for Springer, this for the last 2 items on the original issue